release-tools/travis.yml

```diff
@@ -0,0 +1,19 @@
+language: go
```

Comment:
Should we consider appveyor here given the focus on Windows? Example usage in containerd: https://github.com/containerd/containerd/blob/master/.appveyor.yml

Reply:
I would be open to it.  Can we get someone dedicated to setup the build/test infra for this project?

As a side note, we're also trying to move completely to prow and not use travisci anymore, so we may need to sync up with them on Windows support

Reply:
@wk8 will be looking into it. He is applying for membership to kubernetes-csi for easy tagging.

Reply:
@msau42 : no travis also means no 3rd-party CI at all? No appveyor either?

Reply:
We should check with test-infra on their plans to support Windows in prow.  If it's not anytime soon, then I am open to using a 3rd party